spec/index.html

```diff
@@ -442,42 +444,43 @@ <h3>Quoted Literals</h3>
+        each having the code point <code class="codepoint">U+005E</code>.
+        A datatype IRI in Turtle may be written using either a
+        <a href="#sec-iri">resolved IRI</a>,
+        a <a href="#relative-iri">relative IRI reference</a>,
+        or a <a href="#prefixed-name">prefixed name</a>.
         If there is no datatype IRI and no language tag,
         the datatype is <code>xsd:string</code>.
       </p>
 
-      <p>A backslash ('<code>\</code>', <span class="codepoint">U+005C</span>)
+      <p>A '<code>\</code>' (backslash, code point <code class="codepoint">U+005C</code>)
         may not appear in any quoted literal except as part of an escape sequence.
         Other restrictions depend on the delimiter:</p>
 
       <ul>
-        <li>Literals delimited by <code>'</code> (<span class="codepoint">U+0027</span>)
-          may not contain unescaped <code>'</code>,
-          <code title="LINE FEED"><sub>LF</sub></code> (<span class="codepoint">U+000A</span>),
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code> (<span class="codepoint">U+000D</span>)
+        <li>Literals delimited by <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>)
+          may not contain unescaped
+          <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
-        <li>Literals delimited by <code>"</code>, may not contain unescaped
-          <code>"</code>,
-          <code title="LINE FEED"><sub>LF</sub></code>,
-          nor <code title="CARRIAGE RETURN"><sub>CR</sub></code>
+        <li>Literals delimited by <code>&quot;</code>, may not contain unescaped
+          <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>),
+          <code>LF</code> (line feed, code point <code class="codepoint">U+000A</code>), nor
+          <code>CR</code> (carriage return, code point <code class="codepoint">U+000D</code>)
           characters.</li>
-        <li>Literals delimited by <code>'''</code> may not contain the sequence of characters
-          <code>'''</code>.</li>
+        <li>Literals delimited by a sequence of three <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>) may not contain a sequence of three <code>'</code> (apostrophe, code point <code class="codepoint">U+0027</code>).</li>        
+        <li>Literals delimited by a sequence of three <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>) may not contain a sequence of three <code>&quot;</code> (quotation mark, code point <code class="codepoint">U+0022</code>).</li>
         <li>Literals delimited by <code>"""</code> may not contain the sequence of characters
           <code>"""</code>.</li>
```

Comment:
Removing redundant bullet. Adjusting newer text to match even newer suggestion by @afs.
```suggestion
        <li>Literals delimited by a sequence of three <code>&quot;</code> (quotation
        mark, code point <code class="codepoint">U+0022</code>), i.e.,
        <code>&quot;&quot;&quot;</code>, may not contain such a sequence.</li>
```